clang/test/CheckedC/inferred-bounds/multiple-widened-bounds.c

```diff
@@ -0,0 +1,568 @@
+// Tests for datafow analysis for bounds multiple widening of _Nt_array_ptr's.
```

Comment:
Re-phrase:
"Tests for datafow analysis for bounds widening in case of multiple dereferences of _Nt_array_ptr's."